tests/integration/test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+from copy import deepcopy
+
+import pytest
+import wazuh_testing.vulnerability_detector as vd
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.file import truncate_file, read_json_file, write_json_file, write_file
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools.utils import lower_case_key_dictionary_array
+
+# Marks
+pytestmark = [pytest.mark.server, pytest.mark.tier(level=2)]
+
+# Variables
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, '..', '..', 'data')
+configurations_path = os.path.join(test_data_path, 'configuration', 'test_feeds', vd.INVALID_ALAS_FEEDS_CONF)
+custom_alas_json_feed_path = os.path.join(test_data_path, 'feeds', vd.CUSTOM_ALAS_JSON_FEED)
+custom_alas2_json_feed_path = os.path.join(test_data_path, 'feeds', vd.CUSTOM_ALAS2_JSON_FEED)
```

Comment:
Using the same path as the original feed may cause some problems. For example, if we stop the pytest execution in the middle of the file editing, this could cause future errors.
IMHO it should be more secure to change this path for a temporal one, and edit the fixture in order to write the feed in that file.

Reply:
Done in commit 330f641b6fd3cb999e7d4142cfe95ec1854f80ef